ush/config_defaults.yaml

```diff
@@ -1014,6 +1021,21 @@ nco:
   COMIN_BASEDIR: '{{ COMROOT }}/{{ NET }}/{{ model_ver }}'
   COMOUT_BASEDIR: '{{ COMROOT }}/{{ NET }}/{{ model_ver }}'
 
+  #
+  # New additions from RRFS_dev1
+  #
+  NWGES: '{{ OPSROOT }}/nwges'
+  NWGES_BASEDIR: '{{ NWGES }}'
+  RRFSE_NWGES: '{{ OPSROOT }}/nwges'
+  RRFSE_NWGES_BASEDIR: '{{ RRFSE_NWGES }}'
+  ENSCTRL_NWGES: '{{ OPSROOT }}/nwges'
+  ENSCTRL_NWGES_BASEDIR: '{{ ENSCTRL_NWGES}}'
+
+  ENSCTRL_COMIN: '{{ COMIN_BASEDIR}}'
+  ENSCTRL_COMOUT: '{{ COMOUT_BASEDIR}}'
+  ENSCTRL_COMROOT: '{{ ENSCTRL_COMOUT }}'
+  ENSCTRL_COMIN_BASEDIR: '{{ ENSCTRL_COMIN }}'
+  ENSCTRL_COMOUT_BASEDIR: '{{ ENSCTRL_COMROOT}}/{{ envir}}'
```

Comment:
We shouldn't include any RRFSE or ensemble-related variables here as they are out of context of this PR.

Reply:
Removed